<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.524, y1:1, x2:0.490099, y2:0, stop:0 rgba(48, 216, 239, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inicio">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>351</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Imagenes/Ilustración conceptual búsqueda de archivos Vector Premium.jpeg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="Modelos">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>250</y>
          <width>281</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(48, 216, 239);</string>
        </property>
        <property name="title">
         <string>Modelos</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QRadioButton" name="Booleano_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Booleano</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="Vectorial_3">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Vectorial</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="Fuzzy_3">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>30</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Fuzzy</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="Cuerpos">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>320</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(48, 216, 239);</string>
        </property>
        <property name="title">
         <string>Cuerpos</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QRadioButton" name="Cran_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Cran</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="Med_3">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Med</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="Aceptar">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>390</y>
          <width>89</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
background-color:rgb(32, 74, 135);
}</string>
        </property>
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Buscar">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:hover{
background-color:rgb(32, 74, 135);
}</string>
             </property>
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>350</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>350</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
          </property>
          <property name="title">
           <string>Resultados:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>641</width>
             <height>251</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Ubuntu&quot;;</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>639</width>
              <height>249</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </widget>
          </widget>
          <widget class="QPushButton" name="Atras">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>300</y>
             <width>89</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
font: 11pt &quot;Ubuntu&quot;;
}
QPushButton:hover{
background-color:rgb(32, 74, 135);
font: 11pt &quot;Ubuntu&quot;;
}</string>
           </property>
           <property name="text">
            <string>Atrás</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
